Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the include granted scopes to determine if this authorization request should use
+        /// incremental authorization (https://developers.google.com/+/web/api/rest/oauth#incremental-auth).
+        /// If this is provided with the value <c>true</c>, and the authorization request is granted, the 

```

Comment:
[optional]
I think you can replace "If this is provided with the value true..." with "If true..."